src/components/MediaPlayer/MediaPlayer.js

```diff
@@ -104,6 +104,14 @@ export class MediaPlayerComponent {
     }
 
     this.mainControlPannel.play();
+
+    this.showHidePlayer();
+  }
+
+  showHidePlayer() {
```

Comment:
why is it called showHide player if it only shows it?

Reply:
fixed all